commands/servercommandbase.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * @file
+ * @version 0.1
+ * @copyright 2019 CN-Consult GmbH
+ * @author Jens Stahl <jens.stahl@cn-consult.eu>
+ *
+ * License: Please check the LICENSE file for more information.
+ */
+
+namespace PBST\Commands;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Exception;
+
+/**
+ * Base-Class for all IONOS-related server commands.
+ *
+ * We inherit from our own base class instead direct use of symfony command because there are some lines of code
+ * which has to be executed always. All server commmands should use this class for a base class.
+ *  - Implement method configure() with command-specific additions as needed
+ *      - provide a valid name, check/parse of new input arguments, arbitrary setup code before running the command
+ *  - Implement method execute() with what gets done if the command is triggered
+ */
```

Comment:
Please document that the inheriting commands need an argument serverName that means the server that the command targets.